.github/workflows/docker.yml

```diff
@@ -0,0 +1,32 @@
+name: On push to main add a tag to the commit if docker or renv stuff changed
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Tag in case renv or docker files changed
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # OR "2" -> To retrieve the preceding commit.
+
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            dockerfile
+            docker-compose.test.yml
+            test_docker.sh
+            docker
+            renv.lock
+            .github/workflows/docker.yml
```

Comment:
And a similar remark for this GHA: why should a new docker be build after changes in this GHA (that even doesn't use a docker)?